Lib/datetime.py

```diff
@@ -235,9 +251,11 @@ def _wrap_strftime(object, format, timetuple):
                                 s = rest.seconds
                                 u = offset.microseconds
                                 if u:
-                                    zreplace = '%c%02d%02d%02d.%06d' % (sign, h, m, s, u)
+                                    zreplace = '%c%02d%02d%02d.%06d' % (
+                                        sign, h, m, s, u)
```

Comment:
Again, don't apply autoformatters to the entirety of the code; here's it's even worse than adding extra blank lines; it looks like a real change, and does nothing but make reviewing the patch harder. Same comment applies to all the other times this happened (I don't even like the look of the auto-formatted stuff better; going a few characters over 80 looks better than that weirdly assymmetrical `tuple` split across lines).